### ISSUE TITLE
ResizeTo option always errors now, as react always returns `window` as an object

### ISSUE THREAD

**Issue by AskAlice** [inlet/react-pixi#250]
### Description
https://github.com/inlet/react-pixi/blame/7b10107e73c16541712ba63f9ee6f88e6f179b73/src/stage/index.js#L72

This has been erroring since the commit on this line.

### Steps to reproduce

1. add resizeTo in your stage options
2. set resizeTo to `window`, `global.window`, `document.getElementById('root')`, or `document.getRootNode()`
3. watch as line 72 of stage/index.js is hit as it returns an object.
```JSX
const PixiStage = () => {
  const options = {
    resizeTo: document.getElementById('root'), // or window, or global.window, etc
  };

  return (
        <Stage options={options} width={window.innerWidth} height={window.innerHeight}>
           ...
        </Stage>
)}
```
### Additional info

- `@inlet/react-pixi` version:  5.1.4
- `React` version: 16.13.1
- `ReactDOM` version: 16.13.1
- `PIXI` version: 5.3.0

**Comment by ajitid** [comment]
Turns out [the condition](https://github.com/inlet/react-pixi/blame/7b10107e73c16541712ba63f9ee6f88e6f179b73/src/stage/index.js#L71) itself needs to be boolean opposite as `invariant` runs if condition is not met (is falsy).

So 

```js
el !== window && !(el instanceof HTMLElement)
```

needs to be

```js
el === window || el instanceof HTMLElement
```

**Comment by inlet** [comment]
Thanks @ajitid that was indeed the case. Just pushed an update.

Btw, make sure to add `{ autoDensity: true }` if you'd like to enable auto resolution.

**Comment by Xaaq** [comment]
Hi @inlet,

I want to report that in the newest release (that is `5.1.5`), when providing `options` prop to `Stage` without specifying `resizeTo` attibute it prints out warning in the console:
```
index.js:1 Warning: Failed prop type: Invalid prop `resizeTo` of type undefined, expect `window` or an `HTMLElement`.
``` 

In release `5.1.4` it doesn't cause any warnings.

Sample code to reproduce the problem:
```
function App() {
    return <Stage options={{}}></Stage>
}

ReactDOM.render(<App/>, document.getElementById("root"))
```

**Comment by AskAlice** [comment]
I think the prop validation is a bit overkill given it works when you specify it regardless of the warning

**Comment by zelksnug** [comment]
Nah, having prop validation is fine. @inlet missed putting an early return for `undefined` case before calling `invariant`. 

```js
if (el === undefined) {
  return
}
```

👆 It is [present for `view`](https://github.com/inlet/react-pixi/commit/8dab337696ea865d0a67ecd3a5ec1af6ed97b039#diff-ec7947dd9ccc887abc867073e32ae2a7R79) but not for `resizeTo`

**Comment by inlet** [comment]
Thanks @zelksnug, I’ll create a patch with a fix soon

**Comment by inlet** [comment]
I've just released a new patch `v5.1.6`. The prop `resizeTo` now only validates when set. 

Thanks for contributing!
